### PR TITLE
Result

### PR DESCRIPTION
**Pull request by surbhi84**

### CODE REVIEW

File: pages/quizResult/quizresult.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.true {
+  background-color: rgb(143, 241, 143);
+}
```
Comment (rohanmathur91) [inline pages/quizResult/quizresult.css]: use variable for this color

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me

**Review by ianuragsharma — APPROVED** [review]
LGTM

File: pages/quizResult/quizresult.css (hunk 1 above)
Comment (surbhi84) [reply pages/quizResult/quizresult.css]: done

**Pull request merged**